setup.py

```diff
@@ -1,38 +1,47 @@
-import sys
-from setuptools import setup, find_packages
+import codecs
 import os
+import sys
 
-mod_dir = os.path.dirname(__file__)
-with open(os.path.join(mod_dir, 'bambi', 'version.py')) as f:
-    exec(f.read())
+from setuptools import find_packages, setup
+
+PROJECT_ROOT = os.path.dirname(os.path.realpath(__file__))
+README_FILE = os.path.join(PROJECT_ROOT, "README.md")
+VERSION_FILE = os.path.join(PROJECT_ROOT, "bambi", "version.py")
 
-if len(set(('test', 'easy_install')).intersection(sys.argv)) > 0:
-    import setuptools
 
-requirements = ['numpy', 'pandas', 'patsy', 'pymc3', 'statsmodels', 'arviz']
+def get_long_description():
+    with codecs.open(README_FILE, "rt") as buff:
+        return buff.read()
+
+
+with open(VERSION_FILE) as buff:
+    exec(buff.read())
+
+if len(set(("test", "easy_install")).intersection(sys.argv)) > 0:
+    import setuptools
 
 tests_require = []
 extra_setuptools_args = {}
-if 'setuptools' in sys.modules:
-    tests_require.append('nose')
+if "setuptools" in sys.modules:
+    tests_require.append("nose")
     extra_setuptools_args = dict(
-        test_suite='nose.collector',
-        extras_require=dict(
-            test='nose>=0.10.1')
+        test_suite="nose.collector", extras_require=dict(test="nose>=0.10.1")
     )
 
 setup(
     name="bambi",
     version=__version__,
     description="BAyesian Model Building Interface in Python",
-    url='http://github.com/bambinos/bambi',
-    download_url='https://github.com/bambinos/bambi/archive/%s.tar.gz' % __version__,
-    install_requires=requirements,
-    maintainer='Tal Yarkoni',
-    maintainer_email='tyarkoni@gmail.com',
-    packages=find_packages(exclude=['tests', 'test_*']),
-    package_data={'bambi': ["config/*"]},
+    long_description=get_long_description(),
+    long_description_content_type="text/markdown",
+    url="http://github.com/bambinos/bambi",
+    download_url="https://github.com/bambinos/bambi/archive/%s.tar.gz" % __version__,
+    install_requires=["numpy", "pandas", "patsy", "pymc3", "statsmodels", "arviz"],
```

Comment:
could this be read from requirements.txt?

See example in arviz https://github.com/arviz-devs/arviz/blob/cdbe656a0c759c3f0d83e7285b8b7e8ac5992704/setup.py#L17

Reply:
Done!